apps/mobile/src/theme.ts

```diff
@@ -0,0 +1,18 @@
+import { DefaultTheme } from "react-native-paper";
+
+const theme = {
+  ...DefaultTheme,
+  colors: {
+    ...DefaultTheme.colors,
+    primary: "#51C6B5",
```

Comment:
Is there any reason to use rgb for some colors and hexadecimal for others?

Reply:
Oh, I will standardize it